hopsworks-common/src/main/java/io/hops/hopsworks/common/featurestore/utils/FeaturestoreInputValidation.java

```diff
@@ -0,0 +1,85 @@
+package io.hops.hopsworks.common.featurestore.utils;
+
+import com.google.common.base.Strings;
+import io.hops.hopsworks.common.featurestore.FeaturestoreConstants;
+import io.hops.hopsworks.common.featurestore.FeaturestoreEntityDTO;
+import io.hops.hopsworks.common.featurestore.feature.FeatureDTO;
+import io.hops.hopsworks.exceptions.FeaturestoreException;
+import io.hops.hopsworks.restutils.RESTCodes;
+
+import javax.ejb.Stateless;
+import javax.ejb.TransactionAttribute;
+import javax.ejb.TransactionAttributeType;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.regex.Pattern;
+
+@Stateless
+@TransactionAttribute(TransactionAttributeType.NEVER)
+public class FeaturestoreInputValidation {
+  
+  public FeaturestoreInputValidation() {
+  }
+  
+  /**
+   * Verify entity names input by the user for creation of entities in the featurestore
+   *
+   * @param featurestoreEntityDTO the user input data for the entity
+   * @throws FeaturestoreException
+   */
+  public void verifyUserInput(FeaturestoreEntityDTO featurestoreEntityDTO, Boolean sync)
```

Comment:
What is the `sync` parameter for? It's always `false`

Reply:
I can remove it if we always want to check. see other comment.

Reply:
removed it